### PR TITLE
Add periodic nightly release job for ARM64

### PR DESCRIPTION
**Pull request by zhlhahaha**
Signed-off-by: Howard Zhang <howard.zhang@arm.com>

### CODE REVIEW

**Comment by zhlhahaha** [comment]
cc: @rmohr @fgimenez

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml [hunk 1]
```diff
@@ -837,6 +837,90 @@ periodics:
...
+              set -e
+              sleep 120
+          done
+          if [ $retval -ne 0 ]; then
+              echo "push failed!"
+              exit 1
+          fi
+          # build functest currently does not support crossbuild, skip it
+          git show -s --format=%H > _out/commit
+          build_date="$(date +%Y%m%d)"
```
Comment (rmohr) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml:902]: We probably have the same issue on the other build, but if we would run this job close to midnight, the `build_date` could be misleading. How about filling `build_date` at the beginning and using  it above in `DOCKER_TAG` too?

**Review by rmohr — COMMENTED** [review]
One minor remark.

**Comment by rmohr** [comment]
/cc @fgimenez 

/lgtm
/approve

/hold

@fgimenez if you find the time could you start the job for a test? Otherwise I can do it later.

**Comment by fgimenez** [comment]
> @fgimenez if you find the time could you start the job for a test? Otherwise I can do it later.

sure, running here https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/periodic-kubevirt-push-nightly-build-main-ARM64/1412360124999143424

**Comment by rmohr** [comment]
@fgimenez thx. Everything seems to be pushed correctly.

@zhlhahaha I think you can try out the nightly build and update our user-guide with the arm installation details :)

/unhold

**Pull request merged**